build/build.sln

```diff
@@ -2690,7 +2690,7 @@ Global
 		{B309943F-E9E9-482E-93C7-56FA7D58F898} = {1C64EC86-5799-4311-BED7-0286F61A4D50}
 		{3DC04C10-245A-4EF7-ACB6-12BE7D4ECF76} = {B309943F-E9E9-482E-93C7-56FA7D58F898}
 		{BC8C5D49-060C-4375-904C-FE472C757E58} = {3DC04C10-245A-4EF7-ACB6-12BE7D4ECF76}
-		{A44DA03C-DDA0-495A-9168-30CA6BF87CA5} = {83BDC65C-1E0A-482B-886C-52B78B8B9C97}
+		{A44DA03C-DDA0-495A-9168-30CA6BF87CA5} = {E1BA5F74-6863-437D-9AF5-72666F6D25B5}
```

Comment:
Just want to make sure we don't do something like [https://github.com/Microsoft/WinObjC/pull/2503](https://github.com/Microsoft/WinObjC/pull/2503)


---
In reply to: [112072314](https://github.com/Microsoft/WinObjC/pull/2545#discussion_r112072314) [](ancestors = 112072314)